README.md

```diff
@@ -487,7 +487,7 @@ Support this project with a :star:
 
     # Output list of changed files in a JSON formatted 
     # string which can be used for matrix jobs. Example: 
-    # https://github.com/tj-actions/changed-files/blob/main/.github/workflows/matrix-test.yml 
```

Comment:
Hi @KeisukeYamashita this documentation is autogenerated you'll need to modify the [description in the action.yml](https://github.com/KeisukeYamashita/changed-files/blob/fix-bloken-link-to-matrix-example/action.yml#L130) file instead